src/components/Swap/ReverseButton.tsx

```diff
@@ -10,7 +10,7 @@ import Row from '../Row'
 const ReverseRow = styled(Row)`
   left: 50%;
   position: absolute;
-  transform: translate(-50%, -50%);
+  transform: translate(-50%, -60%);
```

Comment:
negative numbers in CSS are always sus - why 60% now?

Reply:
This is necessary because of the new paddings/margins for the Input/Output.
However, in the initial screen, the Token Select touches the ReverseButton and it doesn't look great. Could we consult with design on that and see if it's intended?

Reply:
- `absolute` positions the element on the left-hand side of the screen w/ it's top left (`0`) pixel at the element's natural start point
- `left: 50%` puts it halfway across the screen
- `translate(-50%, __)` moves the element half its own width to the left of where it would fall given the prior `left: 50%` rule
- `translate(__, -60%)` moves the element up 60% of its own height to center it. (This accounts for the slight asymetry introduced by the new bottom margin btwn the in/output elements that allows the `container` color value to separate them.)

here's the now off-center look of `translate(-50%, -50%)`:
<img width="94" alt="image" src="https://user-images.githubusercontent.com/5773490/194987343-2c3bdbda-6d1e-493d-a340-991c48fa35ad.png">

cc @zzmp